pkg/api/platformapi/allocatorapi/vacate.go

```diff
@@ -377,13 +377,16 @@ func fillVacateClusterParams(params *VacateClusterParams) (*VacateClusterParams,
 
 // newMoveClusterParams
 func newMoveClusterParams(params *VacateClusterParams) (*platform_infrastructure.MoveClustersByTypeParams, error) {
+	req := GetVacateRequestByClusterID(params.ClusterID, params.Kind)
```

Comment:
nit: I'll defer to @marclop and @karencfv, but it might be nice to leave a comment here with a little background.  It's probably not immediately obvious why we're doing this to people who haven't traversed the full vacate tree of logic :).

Reply:
Good point @Z3r0Sum .
I'll add appropriate description why we do this.
Thank you!